### DISCUSSION TITLE
Multi-language support for the repository

### DISCUSSION THREAD

**Discussion by SnmzMert** [minio/minio#17932] [Q&A] [answered]
**Multi-language support for the repository**

_Translating the .md files of this repository into different languages can help it reach a wider audience. If you agree to this, I can start with the Turkish language._

**Comment by klauspost** [comment, marked as answer]
Thanks for the offer!

The README isn't really anything but a quick getting started, which may be simplified a lot, since the main docs are on [docs.min.io](https://min.io/docs/minio/kubernetes/upstream/index.html).

To be honest I see it more realistic that the README will mostly be a link to the official documentation at some point in the future. Maybe with some developer specific information - but not aimed at users of the server. I am speculating as we have no specific plans at the moment.

**Reply by ravindk89** [reply]
To add to this:

We've considered translations for our web docs. However, at this time it's not something we are going to pursue. The difficulty of creating (and reviewing) accurately translated docs isn't the main blocker. It's *maintaining* those docs over time, especially given how often our products change.

Given that there are browser-based tools for translating web pages to a preferred language, we would recommend relying on those tools instead.

We do appreciate the offer though!
